app/models/server/models/Roles.js

```diff
@@ -85,6 +85,14 @@ export class Roles extends Base {
 		return this.findOne(query, options);
 	}
 
+	findByUpdatedDate(updatedAfterDate, options) {
+		const query = {
+			_updatedAt: { $gte: new Date(updatedAfterDate) },
+		};
+
+		return this.find(query, options).fetch();
```

Comment:
I think you can put this `fetch` outside the method(use when you call the `findByUpdatedDate`), this way we can call another model methods like `count`, `forEach`, etc